ci/build_image.sh

```diff
@@ -1,42 +1,30 @@
 #!/usr/bin/env bash
```

Comment:
Nitpick: build_image tells me(the reader) that it's all about building the image so it looks a bit funny to have `build_image.sh build`. Maybe renaming the file to `image.sh` would make it neater? `image.sh build` and then `image.sh push`

Reply:
Done :)